<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyStepDialog</class>
 <widget class="QDialog" name="ModifyStepDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modification d'étape</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
   <item row="13" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
     <item>
      <widget class="QPushButton" name="pushButtonColor">
       <property name="text">
        <string>Changer la couleur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonInitColor">
       <property name="text">
        <string>Réinitialiser</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="labelDetail">
     <property name="text">
      <string>Lien vers détail</string>
     </property>
     <property name="buddy">
      <cstring>pushButtonDetail</cstring>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
     <item>
      <widget class="QPushButton" name="pushButtonDetail">
       <property name="text">
        <string>Parcourir...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonInitDetail">
       <property name="text">
        <string>Réinitialiser</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="15" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="labelID">
     <property name="text">
      <string>ID du step</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="ID">
     <property name="text">
      <string>IDDUSTEP</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Nom du step</string>
     </property>
     <property name="buddy">
      <cstring>lineEditName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditName">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Nom de l'étape dans le job&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Affiché dans le composant uniquement si non vide&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="placeholderText">
      <string>Nom de l'étape</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelProgram">
     <property name="text">
      <string>Programme</string>
     </property>
     <property name="buddy">
      <cstring>lineEditProgram</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditProgram">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Nom du programme&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Affiché dans le composant systématiquement&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="placeholderText">
      <string>Nom du programme</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelPrecision">
     <property name="text">
      <string>Précisions facultatives</string>
     </property>
     <property name="buddy">
      <cstring>textEditPrecision</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTextEdit" name="textEditPrecision">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Informations supplémentaires à afficher</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelTechnical">
     <property name="text">
      <string>Prog. technique</string>
     </property>
     <property name="buddy">
      <cstring>checkBoxTechnical</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBoxTechnical">
     <property name="text">
      <string>(Cocher pour représentation en losange)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelCondition">
     <property name="text">
      <string>Condition</string>
     </property>
     <property name="buddy">
      <cstring>textEditCondition</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QTextEdit" name="textEditCondition">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Conditions sur le step (affichées sous la forme d'un coin rouge)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelData">
     <property name="text">
      <string>Cartes Data</string>
     </property>
     <property name="buddy">
      <cstring>textEditData</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QTextEdit" name="textEditData">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Cartes DATA associées à l'étape (affichées sous la forme d'une flèche noire)</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="labelWidth">
     <property name="text">
      <string>Largeur</string>
     </property>
     <property name="buddy">
      <cstring>spinBoxWidth</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="spinBoxWidth">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="labelHeight">
     <property name="text">
      <string>Hauteur</string>
     </property>
     <property name="buddy">
      <cstring>spinBoxHeight</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QSpinBox" name="spinBoxHeight">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="labelColor">
     <property name="text">
      <string>Couleur</string>
     </property>
     <property name="buddy">
      <cstring>pushButtonColor</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelComment">
     <property name="text">
      <string>Infobulle</string>
     </property>
     <property name="buddy">
      <cstring>textEditComment</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QTextEdit" name="textEditComment">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Infobulle affichée au survol du noeud&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Commentaire (affiché sous forme d'infobulle au survol du composant)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBoxAutoAdjust">
     <property name="text">
      <string>Ajuster automatiquement la taille du noeud</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Ajustement auto.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ModifyStepDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ModifyStepDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
